tests/test_initialization.py

```diff
@@ -1264,6 +1264,23 @@ def test_lora_with_bias_incompatible_arguments(self, extra_kwargs):
         with pytest.raises(ValueError, match=msg):
             LoraConfig(target_modules=["linear"], lora_bias=True, **extra_kwargs)
 
+    def test_lora_forbidden_mamba_modules(self):
```

Comment:
The used model is still relatively big, so I think it's better not to use it in the normal unit tests. I'd suggest to move the test to `test_gpu_examples.py`, that way it runs only nightly, on more powerful workers than the default github runners.